drivers/adc/ad4170/ad4170.h

```diff
@@ -0,0 +1,1142 @@
+/***************************************************************************//**
+ *   @file   ad4170.h
+ *   @brief  Header file for the ad4170 driver.
+ *   @author Darius Berghe (darius.berghe@analog.com)
+********************************************************************************
+ * Copyright 2020, 2023(c) Analog Devices, Inc.
```

Comment:
2024?

Reply:
these dates are correct from the point of view of when development was done, work is being published only now...  no idea which one should we pick